webpack/assets/javascripts/react_app/routes/index.js

```diff
@@ -37,12 +38,20 @@ const AppSwitcher = () => {
           path={path}
           key={path}
           {...routeProps}
-          render={props => handleRoute(Component, props)}
+          render={componentProps => handleRoute(Component, componentProps)}
         />
       ))}
       <Route render={handleFallbackRoute} />
     </Switch>
   );
 };
 
+AppSwitcher.propTypes = {
```

Comment:
There are no props used by the AppSwitcher now, can remove the prop-types and default-props

Reply:
Thanks, I removed the prop types.